ci-operator/step-registry/hypershift/mce/kubevirt/baremetalds/disconnected/conformance/hypershift-mce-kubevirt-baremetalds-disconnected-conformance-workflow.yaml

```diff
@@ -0,0 +1,62 @@
+workflow:
+  as: hypershift-mce-kubevirt-baremetalds-disconnected-conformance
+  documentation: |-
+    The HyperShift KubeVirt conformance workflow executes tests against a new ephemeral
+    HyperShift cluster. Administrative access to the control plane is provided
+    via the `KUBECONFIG` environment variable.
+    This is executed in a fully disconencted cluster with
+    mirrored images.
+
+    Note that tests will execute once the control plane is ready, regardless of
+    whether nodes have been requested, have joined, or are available.
+
+    The control plane is hosted on a dedicated standalone OCP cluster running
+    HyperShift in Azure account and is destroyed after tests complete.
+
+    The HyperShift launch capability is currently supported by the HyperShift
+    team.
+
+    Learn more about HyperShift here: https://github.com/openshift/hypershift
+
+    Track HyperShift's development here: https://issues.redhat.com/projects/HOSTEDCP
+  steps:
+    allow_best_effort_post_steps: true
+    allow_skip_on_success: true
+    post:
+    - ref: hypershift-mce-dump
+    - chain: gather-core-dump
+    - chain: hypershift-mce-kubevirt-destroy
+    - chain: baremetalds-ipi-post
+    test:
+    - chain: hypershift-conformance
+    pre:
+    - ref: assisted-baremetal-conf-devscripts
+    - ref: assisted-baremetal-conf-user-data
+    - chain: baremetalds-ipi-pre
+    - ref: hypershift-mce-install-disconnected
+    - ref: hypershift-kubevirt-install-disconnected    # TODO: postpone after metallb and odf
+    - chain: hypershift-kubevirt-baremetalds-metallb  # TODO: make this disconnected
+    - chain: hypershift-kubevirt-baremetalds-odf      # TODO: make this disconnected
+  # - ref: hypershift-kubevirt-install-disconnected
+    - chain: hypershift-mce-kubevirt-create
+    - chain: hypershift-mce-kubevirt-baremetalds-proxy
+    env:
+      TEST_SKIPS: etcd leader changes are not excessive
+      METALLB_OPERATOR_SUB_SOURCE: qe-app-registry
+      LOCAL_STORAGE_OPERATOR_SUB_SOURCE: qe-app-registry
+      DISCONNECTED: "true"
+      PACKET_PLAN: "m3.large.x86"
+      IP_STACK: v4
+      USER_DATA_FILENAME: "user-data"
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
```

Comment:
change to
```
        IP_STACK=v4
        NETWORK_TYPE=OVNKubernetes
        MIRROR_IMAGES=true
        INSTALLER_PROXY=true
        NUM_WORKERS=0
        NUM_MASTERS=3
        MASTER_VCPU=16
        MASTER_MEMORY=81920
        VM_EXTRADISKS=true
        VM_EXTRADISKS_LIST=vda
        VM_EXTRADISKS_SIZE=100G    
```
and you can try [this](https://github.com/openshift-metal3/dev-scripts/blob/master/config_example.sh#L623) E.g:
```
export MIRROR_OLM="xxxxxx,xxxxxx"
```
to install odf and metallb